File: Examples/DotNET/CSharp/Files/DeleteFileFromThirdPartyStorage.cs
using System;
using Com.Aspose.Storage.Api;
using Com.Aspose.Storage.Model;

namespace Files
{
    class DeleteFileFromThirdPartyStorage
    {
        public static void Run()
        {
            // ExStart:1           
            StorageApi storageApi = new StorageApi(Common.APP_KEY, Common.APP_SID, Common.BASEPATH);

            string Path = "testfile.txt";
            string versionId = null;
            string storage = "MyStorageName";            
            byte[] file = System.IO.File.ReadAllBytes(Common.GetDataDir() + Path);
            try
            {
                // Invoke Aspose.Storage Cloud SDK API to upload and delete file
                storageApi.PutCreate(Path, versionId, storage, file);
                storageApi.DeleteFile(Path, versionId, storage);
            }
            catch (Exception ex)
            {
                System.Diagnostics.Debug.WriteLine("error:" + ex.Message + "\n" + ex.StackTrace);
            }
            // ExEnd:1
        }
    }
}

File: Examples/Perl/Files/DeleteFileFromThirdPartyStorage.pl
#ExStart:1
use lib 'lib';
use strict;
use warnings;
use utf8;
use File::Slurp; # From CPAN
use JSON;

use AsposeStorageCloud::StorageApi;
use AsposeStorageCloud::ApiClient;
use AsposeStorageCloud::Configuration;

my $configFile = '../Config/config.json';
my $configPropsText = read_file($configFile);
my $configProps = decode_json($configPropsText);
my $data_path = '../../../Data/';
my $out_path = $configProps->{'out_folder'};

$AsposeStorageCloud::Configuration::app_sid = $configProps->{'app_sid'};
$AsposeStorageCloud::Configuration::api_key = $configProps->{'api_key'};

# Instantiate Aspose.Storage Cloud API SDK
my $storageApi = AsposeStorageCloud::StorageApi->new();

# Invoke Aspose.Storage Cloud SDK API to delete file from third party storage  
my $storage = 'MySorageName';    		                           
my $response = $storageApi->PutCreate(Path => 'testfile.txt', file => $data_path.'testfile.txt', Storage =>$storage);
$response = $storageApi->DeleteFile(Path => 'testfile.txt', Storage =>$storage);
print $response->{'Status'};
#ExEnd:1